umm-spec-lib/resources/json-schemas/variable/umm/v1.9/umm-var-json-schema.json

```diff
@@ -0,0 +1,604 @@
+      "type": "object",
+      "additionalProperties": false,
+      "description": "The index ranges consist of a LatRange and a LonRange.",
+      "properties": {
+        "LatRange": {
+          "description": "The LatRange consists of an index range for latitude.",
+          "type": "array",
+          "items": {
+            "type": "number"
+          },
+          "minItems": 2,
+          "maxItems": 2
+        },
+        "LonRange": {
+          "description": "The LonRange consists of an index range for longitude.",
+          "type": "array",
+          "items": {
+            "type": "number"
+          },
+          "minItems": 2,
+          "maxItems": 2
+        }
+      },
+      "required": ["LatRange", "LonRange"]
+    },
+    "MeasurementIdentifierType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "The elements of this section allow authors to provide community sourced words or phrases to further describe the variable data.",
+      "properties": {
+        "MeasurementContextMedium": {
+          "description": "This element describes the context/medium within which the measurement was made.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 1024
+        },
+        "MeasurementContextMediumURI": {
+          "description": "This element contains the URI for the context/medium.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 4000
+        },
+        "MeasurementObject": {
+          "description": "This element describes the object which was measured.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 1024
+        },
+        "MeasurementObjectURI": {
+          "description": "This element contains the URI for the object which was measured.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 4000
+        },
+        "MeasurementQuantities": {
+          "description": "This element contains the quantity or quantities which was/were measured.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MeasurementQuantityType"
+          },
+          "minItems": 0
+        }
+      },
+      "required": ["MeasurementContextMedium", "MeasurementObject"]
+    },
+    "MeasurementQuantityType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "The elements of this section apply to a measurement name. The measurement name is structured according to the form defined by Scott Peckham. This is: <object>_<quantity>.",
+      "properties": {
+        "Value": {
+          "description": "This element describes the value for the quantity which was measured.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 1024
+        },
+        "MeasurementQuantityURI": {
+          "description": "This element contains the URI for the quantity which was measured.",
```

Comment:
URI can be enforced in schema

Reply:
yes, we could change the type to be a URI, but first I would want to check all of the data, since this field has been around for 5 years.

Reply:
After fixing 5 records that existed in ES and not oracle, and after getting the data and analyzing it, I saw that I could put the uri constraint on this field, so I did.